app/assets/stylesheets/modules/style.scss

```diff
@@ -0,0 +1,88 @@
+@import "variables";
+
+body {
+  background-image: image-url("default.jpg");
+  background-repeat: no-repeat;
+  background-attachment:fixed;
+  background-size: cover;
+  text-align: center;
+  color: $white;
+  height: 100%;
+  width: 100%;
+  font-family: Lato;
+  position: absolute;
+}
+
+.container-wrapper {
+  background-color: rgba( $black, .4 );
```

Comment:
`.4` should be written with a leading zero as `0.4`<br>Expected 0 spaces between parentheses instead of 1